persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -280,3 +280,7 @@ UPDATE User SET team = ? WHERE qualified_id = ?;
 selectNameByMessageId:
 SELECT name FROM User
 WHERE qualified_id = (SELECT Message.sender_user_id FROM Message WHERE Message.id = :messageId AND Message.conversation_id = :conversationId);
+
+countUsers:
+SELECT COUNT(*)
+FROM User WHERE connection_status == 'ACCEPTED';
```

Comment:
The name says `countUsers`, but it only counts accepted connections.

Maybe `getEstablishedConnectionsCount`.

Also, from a product point of view, should it also count known team mates, for example?

Reply:
It should count the number of user contacts.